routers/user.routers.ts

```diff
@@ -1,24 +1,53 @@
 import { Router } from 'express';
 import { getStudentsList } from './user/getStudentsList';
+import { getStudentCV } from './user/getStudentCV';
```

Comment:
I think You should create one generall class like UserController with all this method to retrive data from user. Controller will be respossible for action with user (getOneStudent, getAllStudentsCount ...).

Reply:
Ok. spotakjmy się na głosowym w tym temacie. potrzebuje więcej informacji.